Firestore/Source/Util/FSTDispatchQueue.mm

```diff
@@ -151,6 +153,12 @@ @interface FSTDispatchQueue ()
  */
 @property(nonatomic, strong, readonly) NSMutableArray<FSTDelayedCallback *> *delayedCallbacks;
 
+/**
+ * Flag set while there's an outstanding FSTDispatchQueue operation, used for assertion
```

Comment:
"Flag set while an FSTDispatchQueue operation is currently executing" ?  (since "outstanding" could be interpreted to mean there are queued operations that haven't run yet)